models/__init__.py

```diff
@@ -1,10 +1,11 @@
 from .ensemble import Ensemble
 from torchvision.models import *
```

Comment:
Why do we need here to import * from torchvision.models?

Reply:
That allows any model class defined in `torchvision.models` to use in config files.